addons/mail/static/src/messaging/entity/partner/partner.js

```diff
@@ -37,6 +24,16 @@ function PartnerFactory({ Entity }) {
          */
         static convertData(data) {
             const data2 = {};
+            if ('country' in data) {
+                if (!data.country) {
+                    data2.country = [['unlink-all']];
+                } else {
+                    data2.country = [['insert', {
+                        id: data.country[0],
+                        name: data.country[1],
```

Comment:
We don't handle the distinction between name and display name for country? Not a big deal either way as it's probably the same.

Reply:
For the moment I think it's good to just use `name`, regardless of whether it's a name of display_name.

Reply:
ok